### DISCUSSION TITLE
Problmes with encoding changes

### DISCUSSION THREAD

**Discussion by dodmi** [StardockCorp/Stardock-Software-Localization#42] [Q&A] [answered]
Hi,

your changes to .gitattributes break my local repository:
- After several tries, I removed my repository completely and forked your repository again
- After cloning a local copy, I end up with the errors
![image](https://github.com/StardockCorp/Stardock-Software-Localization/assets/4572946/5571194a-fe95-4403-9b2c-1a9da3ab4370)
the 16 files are marked as modified locally
![image](https://github.com/StardockCorp/Stardock-Software-Localization/assets/4572946/62aef370-20a9-4566-a0eb-dcf9e5f8e8c4)
and the files are garbled (here de.lng of Groupy 2):
![image](https://github.com/StardockCorp/Stardock-Software-Localization/assets/4572946/fcc8c87a-6bb2-4414-854e-da73ab6b520a)

What do I need to adapt in my local config to get it working? So, it's impossible to contribute...

**Comment by dodmi** [comment]
@magnumpci Am I the only one or can you reproduce my issue?
Anyone?

**Comment by magnumpci** [comment]
There are some things I need to look at on this.

I appreciate the feedback.

Sean Drohan
Stardock Product Lifecycle Manager

**Comment by magnumpci** [comment, marked as answer]
We have re-encoded all the lang files for the apps we have translations for (Start11, Groupy, DeskScapes, Multiplicity).  They are now all under the 'Apps' directory:

https://github.com/StardockCorp/Stardock-Software-Localization/tree/main/Apps

I will work on manually pulling down the orphaned pull requests and then manually uploading the changes.  However, that is going to take some time.  If you wanted to pull the changes, paste in your edits, and create a new pull request, that would be most appreicated. 

Sean Drohan
Stardock Product Lifecycle Manager

**Reply by dodmi** [reply]
Thank you, all my changes were already committed, so de.lng for Groupy is up to date and it seems to be fine now.
